### ISSUE TITLE
Investigate Biorxiv S3 bucket for importing XML

### ISSUE THREAD

**Issue by scottaubrey** [elifesciences/enhanced-preprints#34]

**Comment by will-byrne** [comment]
Had a quick look into the s3 bucket and found this in their [docs](https://www.biorxiv.org/tdm)

> The TDM repository is not intended as a source for further redistribution of articles posted on bioRxiv, or their derivatives, nor does it grant others permission to re-host content posted on bioRxiv.

**Comment by scottaubrey** [comment]
We had a look into this S3 bucket and we're not sure it's useful to our current circumstances.

The access pattern it's designed around is text mining, so the layout of the bucket looks something like `Month_Year/uuid.meca` (e.g. `April_2022/328DAA76-858D-4EBF-8F15-CF4A3F6815A0.meca`).

There's nothing in Biorxiv's APi that would connect the doi to UUID, nor are we sure that published data == S3 bucket date.

In other words, to find a single .meca file for an article, we'd need to create our own index for the whole S3 bucket to figure out which .meca is for which article doi.

So, therefore we think the best route would be to work with biorxiv on a better route, something Paul is already working on.
